### PR TITLE
Update fixture data

### LINKED ISSUE #1416: Fix console errors in article
**Describe the bug**
Error messages have appeared in the article recently

**To Reproduce**

1. Go to any article
2. See error

image

**Expected behavior**
These errors are handled correctly

**Screenshots**

**Additional context**

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1416

**Overall change:** Updates fixture data to reflect the state of test

**Code changes:**

Ran
```
curl https://www.test.bbc.co.uk/news/articles/c9rpqy7pmypo.json | jq . > data/test/news/articles/c9rpqy7pmypo.json
curl https://www.test.bbc.co.uk/news/articles/c85pqyj5m2ko.json | jq . > data/test/news/articles/c85pqyj5m2ko.json
curl https://www.test.bbc.co.uk/news/articles/c85pqyj5m2ko.json | jq . > data/prod/news/articles/c85pqyj5m2ko.json
curl https://www.test.bbc.co.uk/news/articles/c9rpqy7pmypo.json | jq . > data/prod/news/articles/c9rpqy7pmypo.json
```

**Lighthouse**
I have lowered the SEO threshold as real data doesnt have a summary, which we use for the description. This causes lighthouse to throw `Document does not have a meta description`. When we update description for the new data source we should up the threshold again.

Testing - `npm run dev`
`http://localhost:7080/news/articles/c9rpqy7pmypo`
`http://localhost:7080/news/articles/c85pqyj5m2ko`
Only errors should be about manifest.json and sw.js

Testing - `npm run build && npm start`
`http://localhost:7080/news/articles/c9rpqy7pmypo`
`http://localhost:7080/news/articles/c85pqyj5m2ko`
No errors expected

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[ ] Tests added for new features~
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: data/prod/news/articles/c9rpqy7pmypo.json [hunk 1]
```diff
@@ -1,877 +1,923 @@
...
-                  "width": 1024,
-                  "height": 576,
-                  "locator": "A933/production/_101651334_bouquet_pa.jpg",
-                  "originCode": "cpsprodpb",
-                  "copyrightHolder": "PA"
-                }
+                  "copyrightHolder": "PA",
+                  "height": 360,
+                  "locator": "0a88/test/5fe2b600-ccaa-11e8-b0bf-f33155223fc4.jpg",
+                  "originCode": "cpsdevpb",
```
Comment (ChrisBAshton) [inline data/prod/news/articles/c9rpqy7pmypo.json:42]: Just want to make absolutely sure we've updated from the correct env - this looks like test data...

File: data/prod/news/articles/c9rpqy7pmypo.json [hunk 2]
```diff
@@ -1,877 +1,923 @@
...
         {
-          "type": "image",
           "model": {
             "blocks": [
               {
-                "type": "rawImage",
                 "model": {
-                  "width": 1024,
-                  "height": 576,
-                  "locator": "A933/production/_101651334_bouquet_pa.jpg",
```
Comment (ChrisBAshton) [inline data/prod/news/articles/c9rpqy7pmypo.json:131]: ...this looks like prod data

**Review by ChrisBAshton — COMMENTED** [review]
Looks ok (and looks as though Ares have put their data through a `.reverse()`!) but just want to double check we've pulled from the right env...? 🤔

File: data/prod/news/articles/c9rpqy7pmypo.json (hunk 1 above)
Comment (dr3) [reply data/prod/news/articles/c9rpqy7pmypo.json:42]: These assets dont exist on 'prod', theyre test assets. We only have them in 'prod' as we didnt have live assets before. Leaving them here though as theyre useful locally.

File: data/prod/news/articles/c9rpqy7pmypo.json (hunk 2 above)
Comment (dr3) [reply data/prod/news/articles/c9rpqy7pmypo.json:131]: See other comment

File: data/prod/news/articles/c85pqyj5m2ko.json [hunk 3]
```diff
@@ -1,756 +1,801 @@
...
-                "type": "rawImage",
                 "model": {
-                  "width": 1024,
-                  "height": 576,
-                  "locator": "E7DB/production/_101655395_paulineclayton.jpg",
-                  "originCode": "cpsprodpb",
-                  "copyrightHolder": "St Joseph's Hospice"
-                }
+                  "copyrightHolder": "St Joseph's Hospice",
+                  "height": 360,
```
Comment (sareh) [inline data/prod/news/articles/c85pqyj5m2ko.json]: Why has the image height and width been changed?

Comment (dr3) [reply data/prod/news/articles/c85pqyj5m2ko.json]: /shrug :P this is just wants on test

**Review by ChrisBAshton — APPROVED** [review]
Looks good. Thanks Drew. Explanations make sense too

File: package.json [hunk 4]
```diff
@@ -39,7 +39,7 @@
     "test:e2e:storybook": "npm run killStorybook && run-p --race storybook cypress:storybook",
     "test:e2e:storybook:ci": "run-p --race storybook:ci cypress:storybook",
     "test:e2e:storybook:interactive": "npm run killStorybook && run-p --race storybook cypress:storybook:interactive",
-    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80",
+    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=70 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=70",
```
Comment (sareh) [inline package.json:42]: I've opened up another PR to address the meta description issue, so we won't need to reduce the seo threshold. https://github.com/bbc/simorgh/pull/1423

**Review by sareh — APPROVED** [review]
👍 Looks good.

File: package.json (hunk 4 above)
Comment (ChrisBAshton) [reply package.json:42]: 👍 nice. Have put a `blocked` on that PR so that we merge this one first and can re-enable the higher threshold in #1423.

**Comment by jamesbrumpton** [comment]
👍

**Pull request merged**